src/js/state/Search/types.js

```diff
@@ -3,15 +3,14 @@ import type {DateTuple} from "../../lib/TimeWindow"
 import type {SEARCH_BAR_SUBMIT} from "../SearchBar/types"
 import type {Span, Ts} from "../../brim"
 
-export type SearchState = {
+export type SearchState = {|
```

Comment:
In flow the {||} syntax means an "exact" object with only and all these properties. The default {} syntax means "at least" these properties.